libs/langchain/langchain/document_loaders/docugami.py

```diff
@@ -145,8 +146,8 @@ def _create_doc(node: Any, text: str) -> Document:
             """Create a Document from a node and text."""
             metadata = {
                 XPATH_KEY: _xpath_for_chunk(node),
-                DOCUMENT_ID_KEY: document["id"],
-                DOCUMENT_NAME_KEY: document["name"],
+                DOCUMENT_ID_KEY: document[DOCUMENT_ID_KEY],
+                DOCUMENT_SOURCE_KEY: document[DOCUMENT_NAME_KEY],
```

Comment:
technically a breaking change – maybe we include both "name" and "source"?

Reply:
Done, thanks!